composer.json

```diff
@@ -15,7 +15,8 @@
         "ext-zlib": "*",
 
         "robrichards/xmlseclibs": "^3.0",
-        "psr/log": "~1.0"
+        "psr/log": "~1.0",
+        "vimeo/psalm": "^3.0"
```

Comment:
This was an accident, right? 😛

Reply:
Yes, I have psalm installed locally so I can easily run tests.. I will revert this in my next push